lib/uplinkc/testdata/helpers2.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
file name is a little bit lame :)

Reply:
Yea - we'll change it after we get rid of `helpers.h` in the cbindings branch.

Reply:
Let's make it `helper3.c`... :)

Reply:
we need a `helper2.c` first